log/brimapi/client.go

```diff
@@ -0,0 +1,103 @@
+package brimapi
```

Comment:
Consul doens't have anything to do with `brim`, it should be in seperate package